tests/rptest/tests/delete_records_test.py

```diff
@@ -40,11 +42,11 @@ class DeleteRecordsTest(RedpandaTest, PartitionMovementMixin):
     log_segment_size = 1048576
 
     topics = [
-        TopicSpec(name="test-topic-1",
+        TopicSpec(name=TEST_TOPIC_NAME,
```

Comment:
i think if you don't provide a name a randomly generated one will be provided

Reply:
Please take a look at the latest commit. I'm want to use multiple topics in this test, hence the constant introduced here and others introduced later. Referring to topic names by something like `self.topics[1]` is too confusing in my opinion.